src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/help-traditionalApiTokenAccessEnabled.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    Enabling this functionality allows Jenkins API token based access even if the associated user has
+    completly logged out from Jenkins and the OIC Provider.
+
+    The default behavior is to require any Jenkins API token based access to have an valid OIC session user
+    session associated with it.
```

Comment:
I would explicitly mention that the user needs to be logged-in in the UI to be able to use an API Token from the CLI or HTTP API.